sycl/test/abi/layout_handler.cpp

```diff
@@ -186,5 +186,19 @@ void foo() {
 // CHECK-NEXT: 544 |         std::__shared_ptr<class sycl::detail::event_impl, __gnu_cxx::_S_atomic>::element_type * _M_ptr
 // CHECK-NEXT: 552 |         class std::__shared_count<__gnu_cxx::_S_atomic> _M_refcount
 // CHECK-NEXT: 552 |           _Sp_counted_base<(enum __gnu_cxx::_Lock_policy)2U> * _M_pi
-// CHECK-NEXT:     | [sizeof=560, dsize=560, align=8,
-// CHECK-NEXT:     |  nvsize=560, nvalign=8]
```

Comment:
Isn't breaking ABI avoidable?
Tagging @romanovvlad .

Reply:
Yeah, Artur, please avoid breaking ABI. Similar problem is being solved here: https://github.com/intel/llvm/pull/3375